flower_shop/__manifest__.py

```diff
@@ -0,0 +1,36 @@
+{
+    'name': 'Flower Shop ',
+    'version': '1.0',
+    'summary': 'A brief description of your module',
+    'description': 'A longer description of your module',
+    'category': 'Uncategorized',
+    'author': 'ahmad',
+    'website': 'https://www.example.com',
+    'depends': ['base','product','sale_management','website_sale','sale_stock','stock'],
```

Comment:
sale_stock already depends on product and stock, so there is no need to specify those in your dependencies. It should look like this instead:
```'depends': ['base','sale_management','website_sale','sale_stock']```